topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -55,12 +54,11 @@
 	private static final String DEFAULT_BACKGROUND_THEME = "default";
 	private static final String TOP_MENU_BAR_PLACEHOLDER = "tmb_top_menu_bar";
 	private static final String TOP_MENU_ICONS_RIGHT = "icons_right";
-
 	private static final String PORTAL_LINKS = "tmb_portal_links";
 	private static final String TOPICS = "internal_topics";
-
-
 	private static final String COLLAPSEID = "tmb_navbar_collapse";
+	private static String CSS_BLOCK = "TMB_application";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "CSS_BLOCK" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)